src/main/java/org/folio/bulkops/service/ProfileService.java

```diff
@@ -91,13 +116,19 @@ public ProfileDto updateProfile(UUID profileId, ProfileUpdateRequest profileUpda
     if (isLocked || isLockAttempt) {
       validator.checkIfLockPermissionExists();
     }
+    Optional.ofNullable(profileUpdateRequest.getName()).ifPresent(existing::setName);
```

Comment:
Name, entityType and locked are required fields in request. So, I assume, this validation is redundant.